### PR TITLE
1 - API can serve a list of all articles

### PR DESCRIPTION
**Pull request by ViaMarcus**
- Adding article model
- Adding articles controller
- Adding articles index endpoint
- Some testspace config

[PT card](https://www.pivotaltracker.com/story/show/172913618)

### CODE REVIEW

File: app/controllers/api/articles_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Api::ArticlesController < ApplicationController
+  def index
+    @article = Article.all
+    render json: @article
```
Comment (tochman) [inline app/controllers/api/articles_controller.rb]: Ni need to use an instance variable here, and also, I usually call on the Serializer. Are you actually using it?

Comment (kermit-klein) [reply app/controllers/api/articles_controller.rb]: Yes we are using it , it is automatically serializing it we believe
We have a file in initializers folder with the following line in it
```
ActiveModelSerializers.config.adapter = :json
```

Comment (kermit-klein) [reply app/controllers/api/articles_controller.rb]: Instance variable is removed by the way

File: spec/models/article_spec.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```
Comment (erikbjoern) [inline spec/models/article_spec.rb]: This can be refactored into `.rspec`

**Review by erikbjoern — APPROVED** [review]
Everything looks fine now

**Comment by erikbjoern** [comment]
Only question is, if we for readability should call on the serializer.

**Review by kermit-klein — APPROVED** [review]

**Pull request merged**